README.md

```diff
@@ -98,7 +98,8 @@ There’s also a <a href="https://vimeo.com/168648012">fantastic video</a> on ho
 ## Quick start
 
 1. Clone this repo using `git clone --depth=1 https://github.com/mxstbr/react-boilerplate.git`
-1. Run `npm run setup` to install dependencies and clean the git repo.<br />
+1. As an alternative to npm, you can use yarn for dependency management, which might make the installation process easier for different platforms to do so: `USE_YARN=true npm run setup`. If you dont have `yarn` installed check out <a href="https://yarnpkg.com/en/docs/install" target="_blank">here</a> how to install yarn on your OS or do a `npm i -g yarn`
```

Comment:
@sedubois thanks for the squash tip man but github has added a squash feature for the person who merges PR into the target branch within one click :) Since then i prefer not to squash commits so Peer Reviewer can decide himself which message to set and which commits to squash